satellite/marketingweb/server.go

```diff
@@ -112,14 +88,17 @@ func (s *Server) GetOffers(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	offers, err := s.db.ListAll(req.Context())
+	results, err := s.db.ListAll(req.Context())
 	if err != nil {
 		s.log.Error("failed to retrieve all offers", zap.Error(err))
 		s.serveInternalError(w, req, err)
 		return
 	}
 
-	if err := s.templates.home.ExecuteTemplate(w, "base", organizeOffers(offers)); err != nil {
+	var offers rewards.Offers
```

Comment:
Change `ListAll` return type so you don't need to reassign values here

Reply:
That gives me errors from the dbx files. I'm assuming I need to run go generate in `satellite/satellitedb/dbx` and `satellite/satellitedb` again ?